src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/BaseRunner.cs

```diff
@@ -167,7 +187,8 @@ private async Task<(TestResult, string)> RunAsync2(TestCase testCase)
                 } 
                 else if (IsError(result))
                 {
-                    return (TestResult.Pass, null);
+                    // If they override IsError, then do additional checks. 
+                    return await RunErrorCaseAsync(testCase);                    
```

Comment:
This is nice; can we also call the RunErrorCaseAsync for the cases above where the tests passed (lines 155, 164, 176)?

Reply:
We could, but it's less valuable there because we're already returning an ErrorValue. 

It's so special in this case because Sql is returning a blank instead of error; and we want to make sure that it's still generating errors within the expression.